getting_started/step_by_step/resources.rst

```diff
@@ -107,12 +122,23 @@ To obtain an instance of the scene, the method
 :ref:`PackedScene.instance() <class_PackedScene_instance>`
 must be used.
 
-::
+.. tabs::
+ .. code-tab:: gdscript GDScript
 
     func _on_shoot():
             var bullet = preload("res://bullet.tscn").instance()
             add_child(bullet)                  
 
+ .. code-tab:: csharp
+
+    PackedScene _bulletScene = (PackedScene) GD.Load("res://bullet.tscn");
```

Comment:
Same thing for the cast.  
Also if this field could be explicitly `private` that'd be great.